releases.json

```diff
@@ -88,5 +88,18 @@
     "release_date": "2025-06-17",
     "extended_eol": false,
     "security_eol": "2027-02-28"
```

Comment:
what about the EOLs? are they still valid?

Reply:
For now, they should be correct.